app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -284,13 +288,31 @@ def host_subscription_facet(host)
         host.subscription_facet
       end
 
+      def skip_task?(task)
+        # Skip task that doesn't apply errata
+        input = get_task_input(task)
+        input.blank? || input['host'].blank?
+      end
+
+      def get_task_input(task)
+        @_tasks_input[task.id] ||= if task.label == 'Actions::Katello::Host::Erratum::ApplicableErrataInstall'
+          task.execution_plan_action.all_planned_actions(Actions::Katello::Host::Erratum::Install).first.try(:input) || {}
+        else
+          task.input
+        end
+      end
+
       def parse_errata(task)
-        @_tasks_errata_cache[task.id] ||= if task.input['errata'].present?
+        @_tasks_errata_cache[task.id] ||= if get_task_input(task)['errata'].present?
```

Comment:
As part of the katello-ification of katello-agent actions, this task input is no longer actually used.  However i think we should support old and new:

```suggestion
        task_input = get_task_input(task)
        agent_input = task_input['errata'] || task_input['content']
        @_tasks_errata_cache[task.id] ||= if agent_input.present?
                                            # katello agent errata
                                            agent_input
                                          else
```